### PR TITLE
Fifth item complete

### PR DESCRIPTION
**Pull request by SartiMau**
The idea is that the screen automatically refresh itself as soon as a modification is made to the base.

### CODE REVIEW

File: data/src/main/java/com/globant/equattrocchio/data/ImagesRepositoryImpl.java [hunk 1]
```diff
@@ -42,4 +49,40 @@ public void insertImages(List<Image> images) {
...
+    public List<Image> loadAllImages() {
+        Realm realm = Realm.getDefaultInstance();
+
+        RealmResults<ImageEntity> realmImages = realm.where(ImageEntity.class).findAll();
+
+        return transform(realmImages);
+    }
+
+    @Override
+    public Object getAllImageEntities(){
```
Comment (nacho-globant) [inline data/src/main/java/com/globant/equattrocchio/data/ImagesRepositoryImpl.java]: No Object

File: presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/presenter/ImagesPresenter.java [hunk 2]
```diff
@@ -6,29 +6,55 @@
...
+        changeListener = new RealmChangeListener() {
+            @Override
+            public void onChange(Object result) {
+                view.showImagesInCardView(transform((RealmResults<ImageEntity>) result));
+                view.showUpdate();
+                view.showLoadImagesOk();
+            }
+        };
+
+        images.addChangeListener(changeListener);
```
Comment (nacho-globant) [inline presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/presenter/ImagesPresenter.java]: into the repo

File: data/src/main/java/com/globant/equattrocchio/data/ImagesRepositoryImpl.java (hunk 1 above)
Comment (SartiMau) [reply data/src/main/java/com/globant/equattrocchio/data/ImagesRepositoryImpl.java]: done

File: presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/presenter/ImagesPresenter.java (hunk 2 above)
Comment (SartiMau) [reply presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/presenter/ImagesPresenter.java]: done

File: data/src/main/java/com/globant/equattrocchio/data/ImagesRepositoryImpl.java [hunk 3]
```diff
@@ -1,20 +1,23 @@
...
 
 import io.realm.Realm;
+import io.realm.RealmChangeListener;
+import io.realm.RealmResults;
 
 
 public class ImagesRepositoryImpl implements ImagesRepository {
 
     private static final String ID = "id";
+    private RealmResults<ImageEntity> images;
```
Comment (nacho-globant) [inline data/src/main/java/com/globant/equattrocchio/data/ImagesRepositoryImpl.java]: Remove this

**Review by nacho-globant — APPROVED** [review]

**Pull request merged**